src/core/BufferGeometry.js

```diff
@@ -891,6 +891,7 @@ BufferGeometry.prototype = Object.assign( Object.create( EventDispatcher.prototy
 
 			var array = attribute.array;
 			var itemSize = attribute.itemSize;
+			var normalized = attribute.normalized === true;
```

Comment:
It think I would write just:
```js
var normalized = attribute.normalized;
```

Reply:
@Mugen87 I was wondering if I should be consistent with the constructor behavior:
https://github.com/mrdoob/three.js/blob/dev/src/core/BufferAttribute.js#L26

I guess copy as-is makes sense.

Reply:
Oh, okay...

Not a fan of this code style but I see the point^^.

Reply:
(Me neither)
Who can I ask?

Reply:
I believe that's used in the `BufferAttribute` constructor to handle the case where `normalized` is not provided and therefore set to `undefined`. Using `normalized === true` guarantees it's set to a boolean. By the time the attribute is created, though, `attribute.normalized` should be guaranteed to be a boolean value so I don't believe this is needed here.